rules/macos/process_creation/proc_creation_macos_jxa_payoad_execution.yml

```diff
@@ -0,0 +1,29 @@
+title: JXA In-memory Execution Via OSAScript
```

Comment:
No need for this rule. Just add the `.js` extension similar to how you added `.scpt` in #3993 to the same rules in another PR (or simply pull from master and update this)